WLC/__main__.py

```diff
@@ -23,6 +23,7 @@ def arguments():
     parser.add_argument("-d", "--debug", action="store_true", default=False, help="Run in debug mode")
     parser.add_argument("-ip", "--dockerip", help="Docker daemon IP")
     parser.add_argument("-a", "--annotate", action="store_true", default=False, help="Ask user to annotate images")
+    parser.add_argument("-lg", "--language", default='python', help="Which language to execute code in.")
```

Comment:
Change `python` to `python3` maybe? `python2` will be deprecated in 2020.

Reply:
typoed that, fixed